src/System.Data.SqlClient/src/System/Data/SqlClient/SqlConnectionPoolKey.cs

```diff
@@ -50,12 +55,18 @@ internal override string ConnectionString
 
         internal SqlCredential Credential => _credential;
 
+        internal string AccessToken
+        {
+            get
+            {
+                return _accessToken;
+            }
+        }
```

Comment:
This can just be:
```C#
internal string AccessToken => _accessToken;
```
similar to the property right above it.

Reply:
done.